### PR TITLE
Fix for felogin on v12.4.21

### PR DESCRIPTION
**Pull request by lukaszuznanski**

### CODE REVIEW

**Review by soee — APPROVED** [review]
Looks ok.

File: composer.json [hunk 1]
```diff
@@ -49,7 +49,7 @@
     "typo3/coding-standards": "^0.8"
   },
   "conflict": {
-    "typo3/cms-core": "<12.4.3"
+    "typo3/cms-core": "<12.4.20"
```
Comment (bnf) [inline composer.json:52]: Should be <12.4.21

File: Classes/ViewHelpers/LoginFormViewHelper.php [hunk 2]
```diff
@@ -205,13 +205,12 @@ protected function addFieldNamePrefixToViewHelperVariableContainer(): void
...
-     * @param string $name Name
-     *
-     * @return string A hidden field containing the Identity (UID in TYPO3 Flow, uid in Extbase) of the given object or NULL if the object is unknown to the persistence framework
+     * @param mixed $object Object to create the identity field for. Non-objects are ignored.
+     * @param string|null $name Name
+     * @return string A hidden field containing the Identity (uid) of the given object
      * @see \TYPO3\CMS\Extbase\Mvc\Controller\Argument::setValue()
      */
-    protected function renderHiddenIdentityField(?object $object, ?string $name): string
+    protected function renderHiddenIdentityField(mixed $object, ?string $name): string
```
[the remark this answers is not in the document]
Comment (lolli42) [reply Classes/ViewHelpers/LoginFormViewHelper.php:213]: OTOH, mixed plus conflict is ok: after all, 12.4.20 is a security release, and it should be ok not supporting insecure core patch level versions in extensions.

[the remark this answers is not in the document]
Comment (lolli42) [reply Classes/ViewHelpers/LoginFormViewHelper.php:213]: I'm fine with both variants. Please decide on it, looking forward to a headless patch level release, soon!

**Review by lolli42 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (lukaszuznanski) [reply composer.json]: Thanks, fixed

**Pull request merged**